### ISSUE TITLE
Reproducable core dump

### ISSUE THREAD

**Issue by ldelossa** [jovanlanik/gtklock#27]
Played around with this. 

Was unable to catch the core dumps before because daemonizing with "-d" option leaves no place to write out logs. 

```
🖳  gtklock&; sleep 1 && gtklock 
[1] 550298

** (gtklock:550298): WARNING **: 20:17:06.231: gtk-layer-shell v0.7.0 may not work on GTK v3.24.34. If you experience crashes, check https://github.com/wmww/gtk-layer-shell/blob/master/compatibility.md

** (gtklock:550311): ERROR **: 20:17:07.080: Failed to inhibit input. Is another lockscreen already running?
zsh: trace trap (core dumped)  gtklock
```

The above is a common scenario when you have gtklock ran 1) at a timeout and 2) before suspend, where the first time out is earlier then the suspend timeout.

**Comment by jovanlanik** [comment]
I don't understand the issue here? I assume one instance of gtklock stays running?

**Comment by jovanlanik** [comment]
Do you mean the core dump itself? It's caused on purpose by [g_error](https://docs.gtk.org/glib/func.error.html), so it's not really a bug but the docs explicitly don't recommend it for this use case. I'll change it to a message and graceful exit.

**Comment by ldelossa** [comment]
Core dump is usually an indicator that something went pretty wrong. Sorry, I had assumed that gtklock could cause causing some of my issues while I was debugging other things on my desktop. It's because, coredumps will register as pretty large error cases on journalctl output.

**Comment by jovanlanik** [comment]
Yeah, you're right. I shouldn't have used g_error without consulting the documentation.

**Comment by jovanlanik** [comment]
I "fixed" it in master. The solution is a bit clumsy but better that a core dump.

**Comment by ldelossa** [comment]
Thanks
